test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/schema.py

```diff
@@ -441,17 +448,10 @@ def deprecation_schema(for_collection):
     date_schema.update(main_fields)
 
     if for_collection:
-        version_schema = {
-            Required('removed_in'): version(for_collection),
-        }
+        version_schema = {Required('removed_in'): version(for_collection)}
     else:
-        version_schema = {
-            # Only list branches that are deprecated or may have docs stubs in
-            # Deprecation cycle changed at 2.4 (though not retroactively)
-            # 2.3 -> removed_in: "2.5" + n for docs stub
-            # 2.4 -> removed_in: "2.8" + n for docs stub
-            Required('removed_in'): Any("2.12", "2.13", "2.14", "2.15", "2.16"),
-        }
+        version_schema = {Required('removed_in'): deprecation_versions()}
+
```

Comment:
I'd keep the original style here:

```suggestion
        version_schema = {
            Required('removed_in'): deprecation_versions(),
        }
```